tests/helpers.py

```diff
@@ -149,6 +149,20 @@ def create_pod(
     return pod_obj
 
 
+def create_project():
```

Comment:
We have this in OCP class, looks like you added recently

Reply:
I think it's more convenient to have this function in helpers